Modules/_testcapimodule.c

```diff
@@ -2587,6 +2587,55 @@ get_datetime_fromtimestamp(PyObject* self, PyObject *args)
     return rv;
 }
 
+static PyObject *
+test_PyDateTime_GET(PyObject *self, PyObject *obj)
+{
+    int year, month, day;
+
+    year = PyDateTime_GET_YEAR(obj);
+    month = PyDateTime_GET_MONTH(obj);
+    day = PyDateTime_GET_DAY(obj);
+
+    return Py_BuildValue("[lll]", year, month, day);
```

Comment:
[...] is to build a list, previously you built a tuple. Did you use a list on purpose? If not, please use (...) syntax for Py_BuildValue format.
https://docs.python.org/dev/c-api/arg.html#c.Py_BuildValue